docs/HowTo/Monitor/Metrics.md

```diff
@@ -0,0 +1,95 @@
+---
+description: Monitoring and metrics
+---
+
+Enable the [Prometheus](https://prometheus.io/) monitoring and alerting service for
+Teku metrics using the [`--metrics-enabled`](../../Reference/CLI/CLI-Syntax.md#metrics-enabled)
+option.
+
+## Install Prometheus
+
+To use Prometheus with Teku, install the
+[Prometheus main component](https://prometheus.io/download/). On MacOS, install with
+[Homebrew](https://formulae.brew.sh/formula/prometheus):
+
+ ```bash
+ brew install prometheus
+```
+
+!!! tip
+
+    You can also install:
+
+    * Exporters and send system metrics to Prometheus to monitor non-Teku specific items such as
```

Comment:
Exporters 'comma'? trying to understand the intent here...

Reply:
Added a link to the relevant doco